css/at-rules/font-face.json

```diff
@@ -205,22 +207,26 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": false
+                "version_added": true,
```

Comment:
The Chrome status entry says this was a feature "that non WebKit based browsers never implemented" and that sounds about right to me. Let's go with the first non-Presto release for the Operas?

Reply:
Sure thing, coming right up!